test/docfx.Snapshot.Tests/SamplesTest.Seed.Linux/api/toc.html.view.verified.json

```diff
@@ -214,6 +214,7 @@
           "topicHref": "BuildFromProject.Inheritdoc.Issue6366.Class1-1.html",
           "topicUid": "BuildFromProject.Inheritdoc.Issue6366.Class1`1",
           "name.vb": "Inheritdoc.Issue6366.Class1(Of T)",
+          "name.": null,
```

Comment:
I don't know that is related to diffs problems thought.
[CI log](https://pipelinesghubeus23.actions.githubusercontent.com/wvrW66BHwMMAsNx8RDkyegOhUtwVWYJ3kQy5QfKofT6lKKSi7g/_apis/pipelines/1/runs/13208/signedlogcontent/2?urlExpires=2023-12-07T07%3A55%3A54.6202108Z&urlSigningMethod=HMACV1&urlSignature=BiNlhwwvt0jCKGV1smhbvUL9R0EeP5W6XMM8ttPNsao%3D) indicating  `docfx metadata` processing and `dotnet serve` commands seems to be executed in parallel.  
(And ConsoleLogger is registered twice)

> 2023-12-07T07:16:41.1373589Z Serving "/home/runner/work/docfx/docfx/samples/seed/_site" on http://localhost:8089. Press Ctrl+C to shut down.
> 2023-12-07T07:17:04.3573864Z [38;5;15mLoading solution /home/runner/work/docfx/docfx/samples/seed/dotnet/solution/BuildFromSolution.sln[0m
> 2023-12-07T07:17:04.3575573Z [38;5;15mLoading solution /home/runner/work/docfx/docfx/samples/seed/dotnet/solution/BuildFromSolution.sln[0m
> 2023-12-07T07:17:04.3652332Z [38;5;15mLoading project /home/runner/work/docfx/docfx/samples/seed/dotnet/solution/CatLibrary/CatLibrary.csproj[0m
> 2023-12-07T07:17:04.3654210Z [38;5;15mLoading project /home/runner/work/docfx/docfx/samples/seed/dotnet/solution/CatLibrary/CatLibrary.csproj[0m

Reply:
I've compared [CI log](https://github.com/dotnet/docfx/actions/runs/7126025850/job/19403093103) that are successfully completed without diff.
But log output seems same.
So above comment is not related to diff problems.